ui/react-app/src/components/navbar.tsx

```diff
@@ -0,0 +1,49 @@
+import { AppBar, Box, Button, Stack, Toolbar, Typography } from '@mui/material';
+import { useNavigate } from 'react-router-dom';
+
+export default function Navbar(): JSX.Element {
+  const navigate = useNavigate();
+  return (
+    <AppBar position={'fixed'} elevation={1}>
+      <Toolbar
+        sx={{
+          backgroundColor: 'aliceblue',
+        }}
+      >
+        <Box sx={{ display: 'flex', flexDirection: 'row' }} flexGrow={1}>
+          <Button
+            onClick={() => {
+              navigate('/react-app');
+            }}
+          >
+            <Typography
+              variant="h6"
+              sx={(theme) => ({
+                marginRight: '1rem',
+                color: theme.palette.common.black,
+              })}
+            >
+              AlertManager
+            </Typography>
+          </Button>
+          <Button variant="text">Alerts</Button>
+          <Button variant="text">Silences</Button>
+          <Button
```

Comment:
I feel like this one definitely needs an indication when the route is active, unfortunately I couldn't figure out how to combine react-router's `<NavLink />` with mui's `<Button />` without jumping through a few hoops.

We can use the `matchPath` function from `react-router` or build a custom `NavButton` component.

Reply:
> I feel like this one definitely needs an indication when the route is active

By an indication do you mean like the button is deactivated when its route is active ?

Reply:
Yea some visual indication of the active route so users know where they are. You could play around with the `variant` prop or just wrap the contents of the button with `<strong>` so long as there is some visual distinction between an inactive button and an active one.

Reply:
I deactivated the button based on the location. Let me know what do you think about this way :).

Reply:
I'm not sure if using the `disabled` prop is the best way to go about it since it has a certain semantic meaning and users with screen readers might be confused about a navigation item being disabled :)

My suggestion here would be to simply make the text of the button bold but we can move forward with what we have and make another pass at the navigation later since we're only in the very early setup stages here 👍